docs/src/main/sphinx/language/overview.rst

```diff
@@ -0,0 +1,133 @@
+=====================
+SQL statement support
+=====================
+
+The SQL statement support in Trino can be categorized into a number of topics.
+Numerous statements are part of the core engine and therefore available in all
+use cases. For example, you can always set session properties or inspect an
+explain plan and perform other actions with the :ref:`globally available
+statements <sql-globally-available>`.
+
+On the other hand the details and architecture of the connected data sources may
```

Comment:
"On the other hand" seems to be a colloquialism that might not make sense to non-native readers

Reply:
Replaced with "However" ... but I am not too happy with that either. Any other ideas?